graphprot/NeuralNet.py

```diff
@@ -130,43 +146,66 @@ def plot_acc(self, nepoch, train_acc, valid_acc=[]):
                 plt.savefig('acc_epoch.png')
                 plt.close() 
 
+    def plot_hit_rate (self, threshold=4):
```

Comment:
I find the code to get the hitrate a bit confusing. for example I don't know what sorted does on a zip ?  I think to get the hitrate we ususally use something like : 

```python
idx = np.argsort(prediction)
hitrate = np.cumsum(ground_truth[idx])
```
if you have a target where the higher the score the better (like FNAT) you simply invert idx : `idx = idx[::-1]`

I would also split that function in two with a `compute_hitrate(pred, ground_truth)` that doesn't have to be in `NeuralNet.py`  and called that function in `plot_hit_rate()`